phase2/Kconfig

```diff
@@ -15,6 +15,14 @@ config phase2.provider
 
 	  Valid options are (ignition, kubeadm).
 
+config phase2.proxy_mode
+	string "kube-proxy mode: userspace, iptables or ipvs"
```

Comment:
I'm not very familiar with kube-proxy modes, but this configuration seems to imply that kubernetes-anywhere is supporting three different modes for the proxy. Looking at how this field is used, it looks like we only honor a binary decision: ipvs or non-ipvs. I don't know what mode kubeadm uses by default if you don't enable ipvs, but it seems like we should only advertise those two values here (whether that's userspace or iptables, I'm not sure).

Reply:
Thanks for replaying.
Yeah, it was suppose to support those three modes, but for some reason kubeadm only support iptables and ipvs for now. Use iptables by default and use ipvs if we set feature gate `SupportIPVSProxyMode=true`. 
This is short-term solution, and they were planing use kube-proxy ComponentConfig in kubeadm, which will definitely solve this problem.
There are more information in this pr : kubernetes/kubernetes#53962

Reply:
Sure, in that case I would just remove `userspace` from the list for now so it doesn't mislead users.

Reply:
Thanks, should I do that?

Reply:
Oh, just here in the KConfig file. These values aren't enforced by the configuration system, but are used as user-prompted documentation as someone sets up their configuration. If you want to see it in practice, you can step into our Docker dev container with `make docker-dev` and then run `make menuconfig` inside it to use the curses-based frontend. Your configuration from KConfig is shown to the user when they set these fields.

TL;DR: just change your KConfig lines 19 and 24. Right now, they list all three options, but should only list the two that are supported right now.

Reply:
Thanks!